cmad/objectives/objective.py

```diff
@@ -5,13 +5,13 @@
 
 class Objective():
 
-    def __init__(self, qoi, sensitivity_type):
+    def __init__(self, qoi, sensitivity_type, weights=None):
```

Comment:
I want to treat calibration weights more like the data class in terms of their shape/values and evaluation per step in the QoI class. That way they can be specified up front with the data.